migrations/updateMigrations.cmd

```diff
@@ -1,18 +1,64 @@
+SET version=2.1.0-rc1
```

Comment:
@ajcvickers this is the only place that matters now for the version

Reply:
That would have saved a bit of time. :-)